driver/driver.feature

```diff
@@ -376,14 +376,18 @@ Feature: TypeDB Driver
       """
     Then answer type is: concept rows
     Then answer type is not: ok
-    Then answer type is not: concept trees
+    Then answer type is not: concept documents
     Then answer unwraps as concept rows
     Then answer query type is: read
     Then answer query type is not: schema
     Then answer query type is not: write
     Then answer size is: 1
     Then answer column names are:
       | p |
+    Then answer get row(0) query type is: read
```

Comment:
`answer query type is` is to check the `QueryAnwer::getQueryType()`
`answer get row(0) query type` is to check `QueryAnswer::asConceptRows::getQueryType()` (after the consumption)
We don't have it for the documents for many drivers as they receive the string representation of jsons. Not a big issue for now.